gatsby-site/src/components/About.js

```diff
@@ -0,0 +1,65 @@
+import React from "react"
+import PropTypes from "prop-types"
+import styled from "styled-components"
+
+const Section = styled.section`
+  position: relative;
+  overflow: hidden;
+`
+
+const Heading = styled.h2`
+  font-style: normal;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 1.25;
+  letter-spacing: 0.14em;
+  text-transform: uppercase;
+  color: var(--sunrise-gray);
+  padding-bottom: 16px;
+  margin: 0;
+`
+
+const Body = styled.div`
+  font-family: var(--sunrise-font-serif);
+  font-style: normal;
+  font-weight: normal;
+  font-size: 18px;
+  line-height: 1.5;
+  color: #000;
+`
+
+const Paragraph = styled.p`
+  &:last-child {
+    margin-bottom: 0;
+  }
+`
+
+/**
+ * @typedef {Object} AboutProps
+ * @property {string} content
+ * @property {string} hubName
+ *
+ * @param {AboutProps} props
+ */
+const About = ({ hubName, content }) => {
```

Comment:
What're the pros/cons of const vs function?

Reply:
In the context of a react component the differences between const and function don't really matter. const is just my habit. If you'd prefer to use function I'd be happy to change it.